custom_components/abbfreeathome_ci/strings.json

```diff
@@ -72,6 +72,20 @@
         "name": "Window/Door"
       }
     },
+    "cover": {
+      "attic_window_actuator": {
+        "name": "{channel_name} Attic Window"
+      },
+      "awning_actuator": {
+        "name": "{channel_name} Awning"
+      },
+      "blind_actuator": {
+        "name": "{channel_name} Blind"
+      },
+      "shutter_actuator": {
+        "name": "{channel_name} Shutter"
+      }
+    },
```

Comment:
Are the translations required for this one? Are the channel_name’s not unique? I’d would have thought they were but I could be wrong.

If I called my blinds in Free@Home “Living Room Blinds” I imagine they’ll come through in Home Assistant as “Living Room Blinds Blind”.

Reply:
Also, to add, looking at the example here: https://github.com/kingsleyadam/local-abbfreeathome/pull/79

I can see the channel name is unique from the device name. In this case I’d say the translations can be removed and we just use the channel name.